### PR TITLE
Added discord link

### PR DESCRIPTION
**Pull request by M4ximumPizza**
Its a good idea to add Discord link to the mod just in case someone finds this mod before the discord server it self.

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "description": "A Redstone Quality of Life mod. To improve efficiency and productivity.",
   "contact": {
     "sources": "https://github.com/RedstoneQoL/mod",
-    "issues": "https://github.com/RedstoneQoL/mod/issues"
+    "issues": "https://github.com/RedstoneQoL/mod/issues",
+    "discord": "https://discord.gg/wAu9k2fGyK"
```
Comment (Matthias1590) [inline src/main/resources/fabric.mod.json:11]: If you could use the non expiring invite instead that'd be great (https://discord.gg/nXWmrAsJQ8)

**Review by Matthias1590 — CHANGES_REQUESTED** [review]

**Comment by Matthias1590** [comment]
Also please merge into dev instead of main

**Comment by RUGMJ** [comment]
This should go into main as its a change to readme

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (M4ximumPizza) [reply src/main/resources/fabric.mod.json]: It has been updated!

**Review by Matthias1590 — APPROVED** [review]

**Pull request merged**